### PR TITLE
Slack improvement

### PR DESCRIPTION
**Pull request by SYN-tactic**
Give the application the ability to post a slack message as part of the process of cutting a release.

Can post a slack message that looks like this: 
image

### CODE REVIEW

File: src/commands/test/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+const DESCRIPTION = 'Use this command as a template or to test things';
+
+const OPTIONS = [
+  {
+    name: '--testing',
+    description: `With this option nothing happens`,
+  },
+];
+const action = async (type, options) => {
+  await sendReleaseMsgToSlack({
```
Comment (guilhermeKodama) [inline src/commands/test/index.js:15]: niceeee! -1 thing for us to do 😬 🎉

File: src/commands/cut/createSlackMessage.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  ghLink,
+  psLink,
+  deployments = defaultDeployments,
+  channelId = SLACK_TEST_CHANNEL,
+}) => {
+  await sendSlackMessage({
+    id: channelId,
+    text: 'This is a test message',
+    blocks: [
+      buildDatePickerRow({
```
Comment (guilhermeKodama) [inline src/commands/cut/createSlackMessage.js:33]: that's awesome!

File: src/commands/cut/index.js [hunk 3]
```diff
@@ -58,6 +61,13 @@ const action = async (type, options) => {
 
   await createPR(nextVersion);
   await createGithubDraftRelease(nextVersion);
+  // await sendReleaseMsgToSlack({
```
Comment (guilhermeKodama) [inline src/commands/cut/index.js:64]: Let's uncomment and put it to test 😛

**Review by guilhermeKodama — APPROVED** [review]

File: src/commands/cut/index.js (hunk 3 above)
Comment (SYN-tactic) [reply src/commands/cut/index.js:64]: haha sure!

**Pull request merged**